frontend/tests/pages/oidcDebuggerPage.ts

```diff
@@ -0,0 +1,49 @@
+﻿import {BasePage} from './basePage';
+import {expect, type Page} from '@playwright/test';
+
+export class OidcDebuggerPage extends BasePage {
+  static readonly redirectUrl = 'https://oidcdebugger.com/debug';
+  static readonly clientId = `oidc-debugger`;
+  static readonly scopes = `openid profile`;
+
+  debuggerPage: BasePage = new BasePage(this.page, this.page.getByText('Success!'), OidcDebuggerPage.redirectUrl);
```

Comment:
I know the url path is `/debug`, but:
1) it's confusing that the `OidcDebuggerPage` has a  `debuggerPage`
2) fortunately, it's actually the success page, so we can safely call it that and make the code less confusing
So, I'd suggest:
```suggestion
  successPage: BasePage = new BasePage(this.page, this.page.getByText('Success!'), OidcDebuggerPage.redirectUrl);
```

And we can rename `waitForDebuggerPage()` too.